pkg/costmodel/cluster.go

```diff
@@ -40,7 +41,7 @@ const (
 // TODO move this to a package-accessible helper
 type PromQueryContext struct {
 	Client         prometheus.Client
-	ErrorCollector *util.ErrorCollector
+	ErrorCollector *errors.ErrorCollector
```

Comment:
Complete nit, but might be nice if this is just called `errors.Collector` to avoid naming stutter.